### PR TITLE
Fixing RID exception

### PR DESCRIPTION
**Pull request by filipi87**
Fixing the error when invoking SetRtpEncodingParameters
```
[ERROR] Handler::SetRtpEncodingParameters() | throwing MediaSoupClientError: Attempted to change RID values in the encodings.
```

### CODE REVIEW

File: src/Handler.cpp [hunk 1]
```diff
@@ -537,10 +537,17 @@ namespace mediasoupclient
...
+        {
+            auto oldEncoding = parameters.encodings[i];
+            parameters.encodings[i] = encodings[i];
+            // There are a couple of paramters that we are not allowed to change
+            // So we need to preserve this parameter before we change the encoding
+            // More info can be found inside webrtc, media_engine.cc => CheckRtpParametersInvalidModificationAndValues
+            parameters.encodings[i].rid = oldEncoding.rid;
+            parameters.encodings[i].ssrc = oldEncoding.ssrc;
+            parameters.encodings[i].active = oldEncoding.active;
+        }
```
Comment (aconchillo) [inline src/Handler.cpp]: nit: apply `clang-format` to the file.

File: src/Handler.cpp [hunk 2]
```diff
@@ -537,10 +537,17 @@ namespace mediasoupclient
...
-		for (int i = 0; i < parameters.encodings.size() && i < encodings.size(); i++)
-		{
-			parameters.encodings[i] = encodings[i];
-		}
+        for (int i = 0; i < parameters.encodings.size() && i < encodings.size(); i++)
+        {
+            auto oldEncoding = parameters.encodings[i];
+            parameters.encodings[i] = encodings[i];
+            // There are a couple of paramters that we are not allowed to change
+            // So we need to preserve this parameter before we change the encoding
```
Comment (aconchillo) [inline src/Handler.cpp]: nit: `these parameters`

Comment (wcarss) [reply src/Handler.cpp]: second small nit: `paramters` -> parameters

**Review by wcarss — APPROVED** [review]
nits aside, this looks good imo!

**Comment by aconchillo** [comment]
LGTM as well! Thank you!

**Review by aconchillo — APPROVED** [review]

File: src/Handler.cpp (hunk 1 above)
Comment (filipi87) [reply src/Handler.cpp]: Fixed.

File: src/Handler.cpp (hunk 2 above)
Comment (filipi87) [reply src/Handler.cpp]: Fixed

**Pull request merged**